blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/PreaggregateConversions.java

```diff
@@ -34,7 +36,7 @@
 public class PreaggregateConversions {
     
     // todo: punt on TTL
-    private static final TimeValue DEFAULT_TTL = new TimeValue(48, TimeUnit.HOURS);
+    private static final TimeValue DEFAULT_TTL = new TimeValue(Configuration.getInstance().getIntegerProperty(TtlConfig.TTL_CONFIG_CONST), TimeUnit.DAYS);
     private static final String NAME_DELIMITER = "//.";
```

Comment:
@goru97 @tilogaat  Are you cool with changing the default from 48 hours (2 days) to 5 days (the default currently in TtlConfig.TTL_CONFIG_CONST).